src/main/msp/msp.c

```diff
@@ -1680,7 +1680,7 @@ case MSP_NAME:
             if (!serialIsPortAvailable(serialConfig()->portConfigs[i].identifier)) {
                 continue;
             };
-            sbufWriteU8(dst, serialConfig()->portConfigs[i].identifier);
+            sbufWriteU8(dst, serialConfig()->portConfigs[i].identifier | SERIAL_UART_INDEX_BITFLAG);
```

Comment:
Maybe use the flag for UART only - it future, it may be used to mean something different for other port types

Reply:
I was going to use the flag in configurator to choose an entirely different value map dependent on the indexing, but we could simply duplicate the values in the existing map and what you suggest could work. I'll investigate.

Reply:
I think I prefer it as an all or nothing flag. It simplifies the implementation in configurator - meaning we use an entirely different mapping to name (and removes risk of incorrect mapping).

Reply:
Ok, it will do and it is really simple,

Personally, I don't like it much:
- I can understand changing protocol to make (resource constrained) firmware simpler. But configurator can adapt easily
- All or nothing flag would be fine it is is single bit per MSP_CF_SERIAL_CONFIG. But bit in identifier is repeated multiple times

After thinking about it a bit, I'd use new identifier range for zero-based uarts - for example start from 130=uart0  (to set bit7). In configurator use two keys for uart1+ (130 -> uart0, 0,131 -> uart1, 1,132-> uart2) and use received value for set. And in MSP set, convert value back to corresponding SERIAL_PORT_UARTx